content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -9,197 +9,81 @@ card:
+ sudo install minikube-linux-amd64 /usr/local/bin/minikube
 ```
 
-Here's an easy way to add the Minikube executable to your path:
+### Debian package
 
 ```shell
-sudo mkdir -p /usr/local/bin/
-sudo install minikube /usr/local/bin/
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+sudo dpkg -i minikube_latest_amd64.deb
 ```
 
-### Install Minikube using Homebrew
-
-As yet another alternative, you can install Minikube using Linux [Homebrew](https://docs.brew.sh/Homebrew-on-Linux):
+### RPM package
 
 ```shell
-brew install minikube
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+sudo rpm -ivh minikube-latest.x86_64.rpm
 ```
 
 {{% /tab %}}
 {{% tab name="macOS" %}}
-### Install kubectl
-
-Make sure you have kubectl installed. You can install kubectl according to the instructions in [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos).
-
-### Install a Hypervisor
-
-If you do not already have a hypervisor installed, install one of these now:
-
-• [HyperKit](https://github.com/moby/hyperkit)
-
-• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-• [VMware Fusion](https://www.vmware.com/products/fusion)
-
-### Install Minikube
-The easiest way to install Minikube on macOS is using [Homebrew](https://brew.sh):
+If the [Brew Package Manager](https://brew.sh/) installed:
 
 ```shell
 brew install minikube
 ```
 
-You can also install it on macOS by downloading a stand-alone binary:
-
-```shell
-curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
-  && chmod +x minikube
-```
-
-Here's an easy way to add the Minikube executable to your path:
+Otherwise, download minikube directly:
 
 ```shell
-sudo mv minikube /usr/local/bin
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64
+sudo install minikube-darwin-amd64 /usr/local/bin/minikube
 ```
 
 {{% /tab %}}
 {{% tab name="Windows" %}}
-### Install kubectl
-
-Make sure you have kubectl installed. You can install kubectl according to the instructions in [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
-
-### Install a Hypervisor
-
-If you do not already have a hypervisor installed, install one of these now:
-
-• [Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_install)
-
-• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-
-{{< note >}}
-Hyper-V can run on three versions of Windows 10: Windows 10 Enterprise, Windows 10 Professional, and Windows 10 Education.
-{{< /note >}}
 
-### Install Minikube using Chocolatey
-
-The easiest way to install Minikube on Windows is using [Chocolatey](https://chocolatey.org/) (run as an administrator):
+If the [Chocolatey Package Manager](https://chocolatey.org/) is installed, use it to install minikube:
```

Comment:
I checked the name for the product. I think this wording works better
```suggestion
If you have the [Chocolatey](https://chocolatey.org/) package manager installed, use it to install minikube:
```
(minor detail, OK to merge without heeding this)